### PR TITLE
Reimplemented ChromaLink Support as per PR #226.

### PR DESCRIPTION
**Pull request by roxaskeyheart**
I couldn't work out how to contribute to an existing pull request so I created a new PR with the previous contents of #226. I also added the references to the base Chroma interface (@Sharparam) and added the ability to get and set specific ChromaLink zones.

I have tested this code in my app with the Razer Chroma HDK and it works however I had to copy paste the code from the local copy of Colore I downloaded last week to a git-managed fork of the develop branch. So I haven't tested it in this new branch, but I checked and don't believe I missed anything.

I will rebuild and test shortly.

### CODE REVIEW

**Comment by roxaskeyheart** [comment]
Tested and seems to be working okay.

File: Corale.Colore/Core/IChromaLink.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+{
+    using System.Collections.Generic;
+
+    using Corale.Colore.Annotations;
+    using Razer.ChromaLink.Effects;
+
+    /// <summary>
+    /// Interface for Chroma Link functionality.
+    /// </summary>
+    public interface IChromaLink
```
Comment (Sharparam) [inline Corale.Colore/Core/IChromaLink.cs]: Should inherit from `IDevice`.

File: Corale.Colore/Razer/ChromaLink/Constants.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    /// <summary>
+    /// Chroma Link constants.
+    /// </summary>
+    public static class Constants
+    {
+        /// <summary>
+        /// Maximum number of LEDs.
+        /// </summary>
+        [PublicAPI]
+        public const int MaxLEDs = 5;
```
Comment (Sharparam) [inline Corale.Colore/Razer/ChromaLink/Constants.cs]: `MaxLeds`, to follow the naming used on other devices (like mouse).

**Comment by Sharparam** [comment]
Nice work 👍

**Pull request merged**